docs/runtimes/application.ipynb

```diff
@@ -82,16 +82,306 @@
   },
   {
    "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
    "source": [
     "Reusing an already built reverse proxy image is done when:\n",
     "- Redeploying a function that built the reverse proxy once and has `application.status.container_image` enriched.\n",
     "- It was already built manually with `mlrun.runtimes.ApplicationRuntime.deploy_reverse_proxy_image()`.\n",
     "\n",
-    "Using one of the above options can help minimize redundant builds and streamline the development cycle. "
-   ],
+    "Using one of the above options can help minimize redundant builds and streamline the development cycle."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "### combining application with serving function\n",
+    "In this example, we deploy a serving function and use it with our application.\n",
+    "</br></br>\n",
+    "<u>Serving creation</u>:</br>\n",
+    "First, we will create our model file and save it as a .py file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# demo, saved in /your/path/add_ten_model.py\n",
+    "from mlrun.serving import V2ModelServer\n",
+    "\n",
+    "class AddTenModel(V2ModelServer):\n",
+    "    def load(self):\n",
+    "        # No actual model to load, just a demo\n",
+    "        pass\n",
+    "\n",
+    "    def predict(self, request):\n",
+    "        input = request['inputs'][0]\n",
+    "        result = input + 10\n",
+    "        return {\"outputs\": [result]}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "Now we will deploy this model as a serving function:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# first we will create a demo model, using V2ModelServer, we will save this file as .py file.\n",
```

Comment:
"# first, create a demo model, using V2ModelServer. Save this file as .py file.\n",

Reply:
I think this comment can be removed entirely